<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hardware Counter Noise Inquisitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="qw_sidebar" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Plot Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_colorbands">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Color Bands</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_plotmode">
            <item>
             <property name="text">
              <string>sum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>max</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Plot Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Thresholds</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Contribution</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="sb_thr_contrib">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Visits</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_thr_visits">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Dimension Groupings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="cb_lump_benchmark">
            <property name="whatsThis">
             <string>Treat the same benchmark with different parameters as one benchmark</string>
            </property>
            <property name="text">
             <string>Benchmark</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_lump_system">
            <property name="whatsThis">
             <string>Treat resource configurations for the same system as one system</string>
            </property>
            <property name="text">
             <string>System</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_lump_resources">
            <property name="text">
             <string>Resources</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_lump_params">
            <property name="whatsThis">
             <string>Combine noisy measurements of all noise types</string>
            </property>
            <property name="text">
             <string>Parameters</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_lump_noise">
            <property name="text">
             <string>Noise Pattern</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QToolBox" name="tb_filters">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="pg_flt_benchmark">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>162</width>
               <height>65</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Benchmark</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4"/>
            </widget>
            <widget class="QWidget" name="pg_flt_system">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>162</width>
               <height>65</height>
              </rect>
             </property>
             <attribute name="label">
              <string>System</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5"/>
            </widget>
            <widget class="QWidget" name="pg_flt_noise">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>162</width>
               <height>65</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Noise Pattern</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6"/>
            </widget>
            <widget class="QWidget" name="pg_flt_metric">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>162</width>
               <height>65</height>
              </rect>
             </property>
             <attribute name="label">
              <string>Metric</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tw_modes">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open experiment</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
